src/complexity_measures/sample_entropy.jl

```diff
@@ -0,0 +1,144 @@
+using DelayEmbeddings
+using Statistics
+using Neighborhood.NearestNeighbors: inrangecount, Chebyshev
+
+export SampleEntropy
+
+"""
+    SampleEntropy(; r::Real = 0.2 * Statistics.std(x), m::Int = 2, τ::Int = 1,
+        metric = Chebyshev()
+    SampleEntropy(x::AbstractVector; m = 2, τ = 1, metric = Chebyshev())
+
+An estimator for the sample entropy complexity measure (Richman & Moorman,
+2000)[^Richman2000], used with [`complexity`](@ref) and [`complexity_normalized`](@ref).
+
+Providing a univariate timeseries `x` to the constructors automatically determines the
+radius `r` as `0.2 * Statistics.std(x)`.
+
+## Description
+
+Sample entropy is defined as
+
+```math
+SampEn(m, r) = \\lim_{N \\to \\infty} \\left[ -\\ln \\dfrac{A^{m+1}(r)}{B^m(r)} \\right],
+```
+
+An *estimator* for sample entropy using radius `r`, embedding dimension `m`,
+embedding lag `τ` is
```

Comment:
```suggestion
An *estimator* for sample entropy using radius `r`, embedding dimension `m`,
embedding lag `τ` is
```

Reply:
In the first equation you don't define `A, B`, whcih cannot be the same as the below A, B becuase they have different m. Furthermore, I honestly see no reason to say the first sentence. We are very obviously providing estimators for quantities in this package, as it is a numerical package, so we might as well start the description with the estimator.